src/v2/view-builder/views/EnrollProfileView.js

```diff
@@ -29,5 +38,21 @@ const Footer = BaseFooter.extend({
 
 export default BaseView.extend({
   Body,
-  Footer
+  Footer,
+  createModelClass (currentViewState, optionUiSchemaConfig, settings) {
+    let modelClass = BaseView.prototype.createModelClass.call(this, currentViewState, optionUiSchemaConfig);
+
+    settings.parseSchema(currentViewState.uiSchema,
+      (schema) => {
+        currentViewState.uiSchema = schema;
+        modelClass = BaseView.prototype.createModelClass.call(self, currentViewState, optionUiSchemaConfig);
+      },
+      (error) => {
+        this.options.appState.trigger('afterError', {
```

Comment:
We are triggering `afterError` here where as we trigger `this.model.trigger('error')` for the preSubmit callback. Should we be consistent ?

Reply:
`currentViewState.uiSchema` is probably different with V1 property schema. Are we fine with it?

Reply:
The model hasn't been created yet, that's the success case.

Reply:
https://github.com/okta/okta-signin-widget/pull/1747/commits/a5231462b7dd1777ff44f8b20e10f482283fc388

Reply:
I meant the first argument to `parseSchema` function, it's probably different between v1 and v2.

Reply:
Yes for this view they are different. 
One difference is the schema is missing min/max lengths and this is a bug tracked in jira - [OKTA-362693](https://oktainc.atlassian.net/browse/OKTA-362693).